tests/sklearn/test_sklearn_models.py

```diff
@@ -1436,98 +1439,96 @@ def test_predict_correctness(
     load_data,
     default_configuration,
     check_is_good_execution_for_cml_vs_circuit,
-    check_circuit_has_no_tlu,
     is_weekly_option,
-    test_subfunctions_in_fhe=True,
     verbose=True,
 ):
-    """Test correct execution, if there is sufficiently n_bits."""
+    """Test prediction correctness between clear quantized and FHE simulation or execution."""
```

Comment:
this test was overly complicated for basically only testing `check_is_good_execution_for_cml_vs_circuit` on a few FHe samples, so I made it simpler + extracted a small part of it with the new `test_linear_models_have_no_tlu` test (see below)

the only thing I removed here is the fact that we were testing `predict(x)` against `predict(x, fhe="disable")`, which I don't think is very relevant